nuxeo/templates/deployment.yaml

```diff
@@ -32,8 +32,13 @@ spec:
 {{- with .Values.podLabels }}
 {{ toYaml . | indent 8 }}
 {{- end }}
-{{- if .Values.podAnnotations }}
       annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
When config/secrets changed and redeployed, the helm chart changes only the configmap/secrets object but deployment is never changed.

This checksum annotation forces redeployment of pods when configmap is changed